### ISSUE TITLE
Installed via Homebrew, unable to configure any parser but vale

### ISSUE THREAD

**Issue by neontapir** [vale-cli/vale#50]
I'm a new user, just having installed Vale via Homebrew. I can't get Vale to use other rulesets except the build-in one. In this example reproducing the error, I'm trying to lint a series of Asciidoc documents. I have my Vale configuration in a .vale file.

▶ which vale
/usr/local/bin/vale

Here's my .vale file:

[*]
BasedOnStyles = proselint

▶ vale dc
{
  "Checks": null,
  "GBaseStyles": [
    "proselint"
  ],
  "GChecks": {},
  "MinAlertLevel": 1,
  "SBaseStyles": {},
  "SChecks": {},
  "StylesPath": "",
  "RuleToLevel": {},
  "Output": "CLI",
  "Wrap": false,
  "NoExit": false,
  "Sorted": false,
  "Normalize": false,
  "Simple": false,
  "InExt": ".txt"
}
▶ vale *.adoc
✔ 0 errors, 0 warnings and 0 suggestions in 6 files.

When I modify my .vale file to read:

[*]
BasedOnStyles = vale

I get:

▶ vale *.adoc
[several items omitted]
 vol4.adoc
 125:11  warning  Consider removing 'Tiny'       vale.Editorializing
 196:47  warning  Consider removing 'excellent'  vale.Editorializing

✖ 7 errors, 75 warnings and 0 suggestions in 6 files.

If vale is in the BasedOnStyles list in my configuration file, I get the 7 errors and 75 warnings shown above. If it's not, I get 0 of each. I've also tried setting StylesPath to a local directory, I've tried some Markdown files, and I've tried configuring it with other parsers than parselint like 18F and TheEconomist, but to no avail.

Is there logging that would help track down the issue?

**Comment by jdkato** [comment]
Hi,

Thanks for trying out Vale!

The first thing to note is that Vale doesn't ship with any of the others styles. They're available as a [separate downloads in the README](https://github.com/ValeLint/vale#reference-styles). Once downloaded, you need to unzip and put them on your `StylesPath`.

For example, you might have

```
my/local/directory
  ├── proselint/
  ├── TheEconomist/
    ...
```
and then in your `.vale` file:

```ini
StylesPath = my/local/directory

[*]
BasedOnStyles = proselint
```

**Comment by neontapir** [comment]
Thank you for your quick reply, @jdkato . At first I saw the same behavior, but then I realized I had a typo in my path string. After correcting that, your instructions worked.

I would expect the program to print a warning that it couldn't find the style specified, rather than reporting no suggestions. That signal is ambiguous, because it could mean that the document passes all the checks, or that none of the checks were run.
